src/session/cookie-utils.ts

```diff
@@ -0,0 +1,72 @@
+/** @description General utils for managing cookies in Typescript. */
+//> Functions
+/**
+ * @function
+ * @param {string} name Name of cookie.
+ * @param {string} value Value of cookie.
+ * @param {number} time Time in seconds.
+ * @description Set cookie.
+ */
+function setCookie(name: string, val: string, time: number) {
+  const date = new Date();
+  const value = val;
+
+  // Set it expire in n(time) seconds
+  date.setTime(date.getTime() + time * 1000);
+  // Set it
+  document.cookie =
+    name + "=" + value + "; expires=" + date.toUTCString() + "; path=/";
+}
+
+/**
+ * @function
+ * @param {string} name Name of cookie.
+ * @returns {string} Value of cookie.
+ * @description Get cookie.
+ */
+function getCookie(name: string) {
+  const value = "; " + document.cookie;
+  const parts = value.split("; " + name + "=");
+
+  if (parts.length == 2) {
+    return parts.pop()?.split(";").shift();
+  }
+}
+
+/**
+ * @function
+ * @param {string} name Name of cookie
+ * @description Delete cookie
+ */
+function deleteCookie(name: string) {
+  const date = new Date();
+
+  // Set it expire in -1 days
```

Comment:
Change to "Set its expire in -1 days"

Reply:
I'd rather write "Set cookie expiry date to -1 days".